tests/unit/test_templatevars.py

```diff
@@ -114,6 +115,13 @@ def _setup_db(tmp_path_factory):
     logfolder.mkdir()
     db_path = logfolder / WorkflowFiles.LogDir.DB
     with CylcWorkflowDAO(db_path, create_tables=True) as dao:
+        dao.connect().execute(
+            rf'''
+                INSERT INTO workflow_params
+                VALUES
+                    ("cylc_version", "{cylc_version}")
```

Comment:
Should this be:

```suggestion
                    ("cylc_version", cylc_version)
```

Reply:
Is this needed, I think the tests pass as it was, I don't think the Cylc version should matter with template variable loading?

Reply:
The reason it matters is that Cylc 7 workflows do not have the `workflow_template_vars` table.

Reply:
This is an f-string after all!